### DISCUSSION TITLE
How to get the Exhaust ROR curve (Probat P series III 5kg)

### DISCUSSION THREAD

**Discussion by AVRURU** [artisan-roaster-scope/artisan#1246] [Q&A] [answered]
Hi,
How to get the Exhaust ROR curve?
I'm using Probat P series III 5kg
Artisan 2.8.4 / window 10

thank you

**Comment by roasterdave** [comment, marked as answer]
There is an explanation how to use symbolic equations to generate an RoR curve in this [Artisan Blog](https://artisan-roasterscope.blogspot.com/2019/11/symbolic-formulas-basics-new-variables.html#more).  Scroll down to **3.2 Computing Rate-of-Rise (RoR)**.  

If you are using the Artisan Machine setup for Probat PIII  the symbolic equation in the example must be modified to use "Y8", which is the symbol for the "Exhaust" value.
`((Y8 - Y8[-1]) / (t - t[-1])) * 60`

Go to Config>> Device>> Extra devices tab.  Add a new Virtual device, label it "Exhaust RoR" under **Label 2**, paste the above equation under **y2(x)** and be sure **LCD 2**, **Curve 2** and **delta Axis 2** are ticked.

![exhaust ror](https://github.com/artisan-roaster-scope/artisan/assets/30009312/2e362389-6732-452f-9bf2-8d7b8c5df707)

**Reply by AVRURU** [reply]
Thank you @roasterdave 
: )
